jsonnet/dashboards/apps/websocket.libsonnet

```diff
@@ -0,0 +1,206 @@
+          prometheus.target('sum(rate(om_mq_reconnect_total{namespace="$namespace", service="mt-websocket"}[10m]))', legendFormat='MQ reconnects'),
+          prometheus.target('sum(rate(om_ws_invalid{namespace="$namespace", service="mt-websocket"}[10m]))', legendFormat='WS invalid'),
+        ],);
+
+      local threads = graphPanel.new(
+        title='Threads cumulative',
+        description='Sum of all threads across all pods',
+        datasource='$datasource',
+      )
+                      .addTargets([
+        prometheus.target('sum(jvm_threads_current{namespace="$namespace",service="mt-websocket"})', legendFormat='current'),
+        prometheus.target('sum by(state)(jvm_threads_state{namespace="$namespace",service="mt-websocket"})', legendFormat='{{ state }}'),
+      ],);
+
+      local memoryPoolAllocation =
+        graphPanel.new(
+          title='Memory pool allocation rate / 10min',
+          description='Memory pool allocation rate, cumulative from all pods',
+          datasource='$datasource',
+        )
+        .addTarget(prometheus.target('sum by(pool)(rate(jvm_memory_pool_allocated_bytes_total{namespace="$namespace", service="mt-websocket"}[10m]))', legendFormat='{{pool}}'));
+
+      local bytes =
+        graphPanel.new(
+          title='Bytes used',
+          description='Cumulative memory usage by all pods',
+          datasource='$datasource',
+          formatY1='bytes',
+        )
+        .addTarget(prometheus.target('sum(jvm_memory_bytes_used{namespace="$namespace",area="heap",service="mt-websocket"})', legendFormat='used bytes'));
+
+
+      local panels = [
+        memory { gridPos: { x: 0, y: 0, w: 6, h: 4 } },
+        cpu { gridPos: { x: 6, y: 0, w: 6, h: 4 } },
+        traffic { gridPos: { x: 12, y: 0, w: 6, h: 4 } },
+        fileSystem { gridPos: { x: 18, y: 0, w: 6, h: 4 } },
+        row.new('Connections') { gridPos: { x: 0, y: 4, w: 24, h: 1 } },
+        activeConnnections { gridPos: { x: 0, y: 5, w: 24, h: 7 } },
+        row.new('Events and frames') { gridPos: { x: 0, y: 12, w: 24, h: 1 } },
+        eventRate { gridPos: { x: 0, y: 13, w: 24, h: 6 } },
+        sentFrames { gridPos: { x: 0, y: 19, w: 24, h: 7 } },
+        row.new('Errors') { gridPos: { x: 0, y: 26, w: 24, h: 1 } },
+        errorRate { gridPos: { x: 0, y: 27, w: 24, h: 6 } },
+        row.new('JVM') { gridPos: { x: 0, y: 33, w: 24, h: 1 } },
+        threads { gridPos: { x: 0, y: 34, w: 24, h: 7 } },
+        memoryPoolAllocation { gridPos: { x: 0, y: 41, w: 24, h: 8 } },
+        bytes { gridPos: { x: 0, y: 49, w: 24, h: 7 } },
+      ];
+
+      dashboard.new(
+        'Websocket',
+        description='Websocket summary',
+        editable=$._config.grafanaDashboards.editable,
+        graphTooltip=$._config.grafanaDashboards.tooltip,
+        refresh=$._config.grafanaDashboards.refresh,
+        time_from=$._config.grafanaDashboards.time_from,
+        tags=$._config.grafanaDashboards.tags.websocket,
```

Comment:
Please use `$._config.grafanaDashboards.tags.k8sApps`. This dashboard in open-source project should be general for any websocket server, so we shouldn't use tags or links related to our use-case.